app/javascript/packs/validators/validators_map.vue

```diff
@@ -109,9 +109,14 @@
     },
     watch: {
     },
-    computed: mapGetters([
-      'network'
-    ]),
+    computed: {
+      leader_valid(){
```

Comment:
For methods which returns a boolean I would suggest the following `is_leader_valid`.

If it was ruby, we would use the `leader_valid?`.